docs/user/dql/functions.rst

```diff
@@ -893,6 +916,33 @@ Example::
     +----------------------+------------------------------------------+
 
 
+DATETIME
+--------
+
+Description
+>>>>>>>>>>>
+
+Usage: datetime(datetime)/ datetime(date, to_timezone) Converts the datetime to a new timezone
```

Comment:
I'd split these and explain what timezone `datetime(datetime)` uses. How does `datetime(datetime)` transforms its parameter?

Also explain when it return null value.